ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -1148,3 +1156,168 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
 }
 
 @end
+
+
+#pragma mark - ORKFormItemLocationCell
+
+@interface ORKFormItemLocationCell () <ORKLocationSelectionViewDelegate>
+
+@property (nonatomic, assign) BOOL editingHighlight;
+@property (nonatomic, strong) NSString *placeholder;
+
+@end
+
+
+@implementation ORKFormItemLocationCell {
+    ORKLocationSelectionView *_selectionView;
+    NSLayoutConstraint *_heightConstraint;
+}
+
+- (void)cellInit {
+    [super cellInit];
+    
+    _selectionView = [[ORKLocationSelectionView alloc] initWithOpenMap:NO
+                                                    useCurrentLocation:((ORKLocationAnswerFormat *)self.formItem.answerFormat).useCurrentLocation
+                                                edgeToEdgePresentation:NO];
+    _selectionView.delegate = self;
+    
+    if (_placeholder != nil) {
+        [_selectionView setPlaceholderText:_placeholder];
+    }
+    
+    [self applyAnswerFormat];
+    [self answerDidChange];
+    
+    [self.contentView addSubview:_selectionView];
```

Comment:
Why not move this to line 1183, then probably you don't need  the ivar `_placeholder`.